src/OpenLoco/Audio/Audio.cpp

```diff
@@ -1103,7 +1103,7 @@ namespace OpenLoco::Audio
             // Choose a track to play, unless we have requested one track in particular.
             if (_currentSong == kNoSong || !requestedSong)
             {
-                auto trackToExclude = _lastSong;
+                uint8_t trackToExclude = _lastSong;
```

Comment:
this is an amusing issue it would be causing next music track to always exclude the current song?!? which may be the source of music issues when changing track.